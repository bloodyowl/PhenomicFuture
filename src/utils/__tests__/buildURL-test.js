const buildURL = require("../buildURL")

describe("buildURL", () => {

  it("should build URLs", () => {
    const toURL = buildURL("ROOT")

    expect(toURL({ url: "/test" })).toMatchSnapshot()
    expect(toURL({ url: "/test", limit: 10 })).toMatchSnapshot()
    expect(toURL({ url: "/test", limit: 10, after: "test" })).toMatchSnapshot()
  })

})
