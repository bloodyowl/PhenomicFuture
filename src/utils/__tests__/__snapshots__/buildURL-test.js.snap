exports[`buildURL should build URLs 1`] = `"ROOT/test.json"`;

exports[`buildURL should build URLs 2`] = `"ROOT/test/10.json"`;

exports[`buildURL should build URLs 3`] = `"ROOT/test/10/test.json"`;
